pkg/client/cli/telepresence_test.go

```diff
@@ -619,6 +637,10 @@ func (cs *connectedSuite) TestJ_ListOnlyMapped() {
 }
 
 func (cs *connectedSuite) TestK_DockerRun() {
+	// This test only runs on linux as it requires docker, and CI can't run linux docker containers inside non-linux runners
+	if goRuntime.GOOS != "linux" {
+		return
+	}
```

Comment:
Prefer to use `cs.SkipNow()` instead of just returning early.

Reply:
So CI has `docker` installed, it's just that pulling the image will fail because there's no windows version available.

I'll make the skipNow change